filebrowser/run.sh

```diff
@@ -21,10 +21,21 @@ if [ "${DB_PATH}" != "filebrowser.db" ]; then
   DB_FLAG=" -d ${DB_PATH}"
 fi
 
+# Check if filebrowser db exists
+if [ ! -f ${DB_PATH} ]; then
+  filebrowser $DB_FLAG config init >> ${LOG_PATH} 2>&1
+  filebrowser $DB_FLAG users add admin "" --perm.admin=true --viewMode=mosaic >> ${LOG_PATH} 2>&1
+fi
+
+filebrowser $DB_FLAG config set --baseurl=${SERVER_BASE_PATH} --port=${PORT} --auth.method=noauth --root=$ROOT_DIR >> ${LOG_PATH} 2>&1
+
+printf "👷 Starting filebrowser in background... \n\n"
+
+
 printf "📂 Serving $${ROOT_DIR} at http://localhost:${PORT} \n\n"
 
-printf "Running 'filebrowser --noauth --root $ROOT_DIR --port ${PORT}$${DB_FLAG} --baseurl ${SERVER_BASE_PATH}' \n\n"
+printf "Running 'filebrowser $DB_FLAG' \n\n"
 
-filebrowser --noauth --root $ROOT_DIR --port ${PORT}$${DB_FLAG} --baseurl ${SERVER_BASE_PATH} > ${LOG_PATH} 2>&1 &
+filebrowser $DB_FLAG >> ${LOG_PATH} 2>&1 &
```

Comment:
I want to make it clear that this issue _was not introduced by this PR_.

But I'll just mention this again; this will break if `DB_PATH` has spaces in it. A better way would be to initialize this as an array and use it in a way that handles spaces:

```bash
args=()
if [ "${DB_PATH}" != "filebrowser.db" ]; then
	args+=(-d "${DB_PATH}")
fi
```

Then use this as:

```bash
filebrowser "${args[@]}"
```

(I did not verify if `$$` vs `$` is required here due to terraform templates.)

This syntax works even in bash-3.2, so should be fine to use. (I opted for arrays as I see we already have a heavy dependency on Bash here.)

Reply:
TBH There's a lot of places that it could break because of spaces in the path, the LOG_PATH and SERVER_BASE_PATH could also cause problems.

I do think I found a bit easier and more readable way to handle it. Filebrowser supports the FB_DATABASE environment variable. So I just set that at the top inside quotes and then it doesn't matter, it works with or without spaces.